modules/gsuite_enabled/outputs.tf

```diff
@@ -36,6 +36,11 @@ output "group_email" {
   description = "The email of the created GSuite group with group_name"
 }
 
+output "group_name" {
+  value       = module.gsuite_group.name
+  description = "The email name of the created GSuite group with group_name"
```

Comment:
I didn't quite use your suggestion, as the GSuite spelling wasn't consistent with the prior output, and your description was maybe confusing about the difference between group_email and group_name, so I went with "The group_name of the GSuite group"

Reply:
We should still fix it to be "G Suite" as that is the official name.

If this output is "my-team" (*not* my-team@my-corp.com) then the proposed description makes sense.

Reply:
Yes, that's correct.  group_email is the full address, group_name is just the username portion of the address, but I wanted to avoid the word username.  I just pushed an update with the spelling correction, too.